cobalt/black_box_tests/black_box_tests.py

```diff
@@ -155,7 +158,9 @@ def CreateCobaltRunner(self, url=None, target_params=None, **kwargs):
         launcher_params=_launcher_params,
         url=url,
         target_params=all_target_params,
-        **kwargs)
+        web_server_port=web_server_port,
```

Comment:
Would it be possible to move the `web_server_port` to be a member of `_launcher_params` so it gets tunneled trough to the runner, without having to modify all callers ?

Reply:
After moving web_server_port into target_params, we will still need to modify all the tests to pass in target_params.

Also, target_params is used to pass in Cobalt flags such as `--preload`. It feels strange to pass in variables for the launcher.